### PR TITLE
New button and seed file

### PR DESCRIPTION
**Pull request by JMD1986**
added a new meeting button to the rooms show page.

also edited the seed file. want to merge it with @wrpaape seed file to make a better seed file.

### CODE REVIEW

File: app/views/meetings/_form.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
   </div>
   <div class="field">
     <%= f.label :room_id %><br>
-    <%= f.select :room_id, @room_options %>
+    <%= f.select :room_id, @room_options, selected: params[:room_id] %>
```
Comment (mlgiardina) [inline app/views/meetings/_form.html.erb:23]: :+1:

File: db/seeds.rb [hunk 2]
```diff
@@ -16,36 +16,35 @@
...
-    end
+    room = Room.create(name: "The #{Faker::Commerce.color.capitalize} Room", max_occupancy: Faker::Number.number(2),
+                room_number: rand(200..400),
+                     imgurl: image.sample,
+                   location: location.sample,
+                    company_id: company.id)
+    amenity = Amenity.create(perk: amenities.sample,
+                             room_id: room.id)
+
+    meeting = Meeting.create(title: Faker::Company.bs, agenda: Faker::Lorem.paragraph,
```
Comment (mlgiardina) [inline db/seeds.rb:41]: I thought we were making "real" agendas?

Comment (JMD1986) [inline db/seeds.rb:41]: I'm working on that right now. I just had to push all this bc the button. If you can find more realistic agenda stuff feel free to pull this and add it.

**Comment by alejandroereyes** [comment]
:+1:

**Pull request merged**